### ISSUE TITLE
Create checklists for onboarding and offboarding members

### ISSUE THREAD

**Issue by nebrius** [nodejs/inclusivity#109]
## Purpose

Now that we're starting to admit new members (sorry for the huge delays everyone!), we should formalize everything that needs to be done to onboard them properly. There are a number of details that need to be covered, and it's proven easy to forget some of the steps.

## Proposal

The onboarding and offboarding lists need to include :

- Inviting people to Slack, if they have not already been invited
- Setting them as an Admin on the Slack
- Inviting them to the members-only channels
- Updating the README and HISTORY documents
- Adding them to the inclusivity@iojs.org mailing list
- Giving them write access to this repository

I feel like I may be forgetting one or two more items, so if you know of any, let me know!

## Focus

This issue is focused purely on codifying the list of actions that need to be taken when onboarding and offboarding members. This means that we want to focus on documenting the existing process only.

Some off topic examples include:
- Changing what rights and responsibilities members or contributors have
- Whether or not we should have private members only channels in Slack
- Whether or not we should be using Slack at all

As always, if you would like to discuss these issues, please open a new issue that follows our [issue policy](https://github.com/nodejs/inclusivity/blob/master/docs/POLICY_ISSUES.md).

**Comment by scottgonzalez** [comment]
I've created a template that we can use to create issues. Every time a new member is accepted, we can create an issue from this template and then work through the list. If this seems good, I can create a PR from this and then we can wait a few days for any other items that we might need to add.

### Template for adding a new member

* [ ] Add them to the members sections of the [README](https://github.com/nodejs/inclusivity/edit/master/README.md) and [HISTORY](https://github.com/nodejs/inclusivity/edit/master/docs/history.md) documents.
* [ ] [Add them to the inclusivity@iojs.org mailing list](https://github.com/nodejs/email/edit/master/iojs.org/aliases.json).
* [ ] [Add them to the Inclusivity GitHub team](https://github.com/orgs/nodejs/teams/inclusivity).
* Slack
  * [ ] Invite them to join Slack.
    * Once #111 is done, just point to the sign up page.
  * [ ] [Set them as an Admin](https://node-inclusivity.slack.com/admin#active).
  * [ ] Invite them to the members-only channels.

### Template for removing a team member

* [ ] [Remove them from the inclusivity@iojs.org mailing list](https://github.com/nodejs/email/edit/master/iojs.org/aliases.json).
* [ ] [Remove them from the Inclusivity GitHub team](https://github.com/orgs/nodejs/teams/inclusivity).
* Slack
  * [ ] [Set them as a Member](https://node-inclusivity.slack.com/admin#active) (can only be done by owners).
  * [ ] Remove them from the members-only channels.

I guessed at the URL for editing the GitHub team. Can someone with access verify that this is correct?

**Comment by nebrius** [comment]
hmm, I would have thought you'd have access? Both URLs worked for me

**Comment by nebrius** [comment]
I think this looks good to PR

**Comment by scottgonzalez** [comment]
> hmm, I would have thought you'd have access?

Oh, look at that; I do have access. I guess that makes sense so you can leave a group on your own. I always assumed you needed to be an admin to get to the teams. I think I just made a bad assumption based on getting access to a single repo's collaborator list, since you need to be an admin to get to the repo settings.
